src/App.js

```diff
@@ -37,7 +37,8 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
+      <Tweet tweet={tweetsArray[0]}></Tweet>
```

Comment:
You did all the components right but you only rendered 1 tweet, you need to render the other two here. 

You can either call the `<Tweet/>` component 3 times or make a loop:
```
function App() {
  return (
    <div className="App">
      <Tweet tweet={tweetsArray[0]}></Tweet>
      <Tweet tweet={tweetsArray[1]}></Tweet>
      <Tweet tweet={tweetsArray[2]}></Tweet>
    </div>
  );
}
```